### DISCUSSION TITLE
Meinheld has been updated to 1.0.2

### DISCUSSION THREAD

**Discussion by Gelmo** [tiangolo/meinheld-gunicorn-docker#130] [Questions] [answered]
Hey taingolo,

Looks like Meinheld got an update shortly after your last push. I need 1.0.2 due to the following:

https://github.com/mopemope/meinheld/pull/102

Please update the Docker Hub image when possible. TIA

**Comment by hingston** [comment, marked as answer]
It has been updated now. This issue can be closed.
